packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                                    {formatTime(
+                                        handleScheduledDeliveryLogs(
+                                            item,
+                                            logs,
+                                        )[0].scheduledTime,
+                                    )}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        sendNotificationLogs(item, logs)[0]
+                                            .scheduledTime,
+                                    )}
+                                </Text>
+                            </Collapse>
+                        </Box>
+                    ) : (
+                        <Text fz={12} color="gray.6">
+                            -
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'deliveryStarted',
+                label: 'Delivery start',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="xxs">
+                                <Text fz={12} fw={500} color="gray.6">
+                                    {formatTime(
+                                        currentLogs(item, logs)[0].createdAt,
+                                    )}
+                                </Text>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        handleScheduledDeliveryLogs(
+                                            item,
+                                            logs,
+                                        )[0].createdAt,
+                                    )}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        sendNotificationLogs(item, logs)[0]
+                                            .createdAt,
+                                    )}
+                                </Text>
+                            </Collapse>
+                        </Box>
+                    ) : (
+                        <Text fz={12} color="gray.6">
+                            -
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'status',
+                label: 'Status',
+                cell: (item) => {
+                    return (
+                        <Stack align="center" justify="center">
+                            {currentLogs(item, logs).length > 0 ? (
```

Comment:
better if you save this above as a variable and reuse below when you pass it to the function